controllers/game.controller.js

```diff
@@ -0,0 +1,137 @@
+const games = [
+    {   
+        gameId: 1,
+        title: "Age of empires",
+        genre: "Strategy",
+        price: 500,
+    }, 
+    {
+        gameId: 2,
+        title: "CSGO",
+        genre: "Strategy",
+        price: 200,
+    }
+]
+
+//[GET] /games
+module.exports.game = (req, res) => {
+    try {
+        const records = games.map(item => item);
+        // console.log(getGame);
+        res.json({
+            code: 200,
+            message: "Success!",
+            data: records,
+        });
+    } catch (error) {
+        res.status(500).json({
+            code: 500,
+            message: "Error",
+            error: error.message
+        })
+    }
+
+}
+
+//[POST] /games
+module.exports.createGame = (req, res) => {
+    try {
+        const newGame = req.body;
+
+        if(newGame.title == "" || newGame.genre == ""){
```

Comment:
Validate a string using trim(). For example, if a user sends the string ' ' (with only spaces), trim() should remove the spaces.